modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -3095,10 +3095,18 @@ private ObjectEntry _updateObjectEntry(
 		if (partialUpdate) {
 			serviceBuilderObjectEntry =
 				_objectEntryService.partialUpdateObjectEntry(
+					_getObjectEntryFolderId(
+						serviceBuilderObjectEntry.getCompanyId(),
+						serviceBuilderObjectEntry.getGroupId(), objectEntry,
+						serviceContext),
```

Comment:
we need to sort the parameters in `_objectEntryService.partialUpdateObjectEntry()`